### ISSUE TITLE
BUG - [Sev - 2] - [Android - Pixel] - Pixel Screen reader does not focus on Back button when navigating to new page - Must swipe to focus on anything

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#13319]
## What happened?

Pixel does not highlight anything when Talkback is turned on when following a link in the app to another section.  When the new page appears, nothing is highlighted.  (Samsung is fine.)  See the recording below.

## Specs:

- Device: Pixel 
- OS: Android 16 - April 5, 2026
- User Account: Demo
- Accessibility State: Talkback ON

## Steps to Reproduce

1. Turn talkback on.
2. From home page, follow a carousel link or a card link. Alternatively, following any links from Health, Benefits, or payments.
3. Notice that nothing is highlighted on the page that opens.  Back should be highlighted.  User must swipe to cause Back to be highlighted.
## Desired behavior

When going to a new page that has Back, it should be highlighted and focused.
## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low 
- Frequency: High  

- 2 - High

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/2dd57bde-0052-4bb9-bdd4-b6238585f65b

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [x] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
